judgels-frontends/raphael/src/routes/uriel/contests/single/submissions/Bundle/ContestSubmissionsPage/ContestSubmissionsPage.tsx

```diff
@@ -0,0 +1,123 @@
+import React, { Component } from 'react';
+import { Card, HTMLTable, H3 } from '@blueprintjs/core';
+import { withRouter } from 'react-router-dom';
+import Pagination from 'components/Pagination/Pagination';
+import { ContestItemSubmissionsResponse } from 'modules/api/uriel/contestSubmissionBundle';
+import { contestSubmissionActions as injectedContestSubmissionActions } from '../modules/contestSubmissionActions';
+import { selectContest } from 'routes/uriel/contests/modules/contestSelectors';
+import { AppState } from 'modules/store';
+import { connect } from 'react-redux';
+import { Contest } from 'modules/api/uriel/contest';
+import { FormattedDate } from 'components/FormattedDate/FormattedDate';
+
+import './ContestSubmissionsPage.css';
+import { ItemSubmission } from 'modules/api/sandalphon/submissionBundle';
+import { push } from 'react-router-redux';
+import { GradingTag } from '../GradingTag/GradingTag';
+
+export interface ContestSubmissionsPageProps {
+  getSubmissions: (
+    contestJid: string,
+    userJid?: string,
+    problemJid?: string,
+    page?: number
+  ) => Promise<ContestItemSubmissionsResponse>;
+  gotoSummary: (contest: Contest, item: ItemSubmission) => any;
+  contest: Contest;
+}
+
+interface ContestSubmissionsPageState {
+  response?: ContestItemSubmissionsResponse;
+}
+
+export class ContestSubmissionsPage extends Component<ContestSubmissionsPageProps, ContestSubmissionsPageState> {
+  private static PAGE_SIZE = 20;
+
+  constructor(props: ContestSubmissionsPageProps) {
+    super(props);
+    this.state = { response: undefined };
+  }
+
+  componentDidMount() {
+    this.loadSubmissions();
+  }
+
+  loadSubmissions = async (page?: number) => {
+    const { contest, getSubmissions } = this.props;
+    const response = await getSubmissions(contest.jid, undefined, undefined, page);
+    this.setState({ response });
+    return response;
+  };
+
+  onChangePage = async (nextPage: number) => {
+    const response = await this.loadSubmissions(nextPage);
+    return response.data.totalCount;
+  };
+
+  render() {
+    const response = this.state.response;
+    if (!response) {
+      return <Card className="bp3-skeleton">{'fake'.repeat(100)}</Card>;
+    }
+
+    const { data, profilesMap, problemAliasesMap } = response;
+    const { contest, gotoSummary } = this.props;
+    const canManage = response.config.canManage;
+
+    return (
+      <Card className="contest-bundle-submissions-page">
+        <H3>Submissions</H3>
+        <HTMLTable className="submissions-table" bordered striped interactive>
+          <thead>
+            <tr>
```

Comment:
I think the order should be:

- User
- Prob
- No
- Answer
- Verdict
- Time

(similar to programming's)